frontend/src/modules/application/actions.js

```diff
@@ -33,10 +33,18 @@ export const createNewApplication = payload => async dispatch => {
 // action used by applicant positions view
 export const fetchPositionsSuccess = payload => ({ type: FETCH_POSITIONS_SUCCESS, payload })
 export const fetchPositions = () => async dispatch => {
-    const res = await fetch("/api/v1/positions")
-    const data = await res.json()
+    try{
+        var res = await fetch("/api/v1/positions")
```

Comment:
We need to send credentials with every `fetch` otherwise it will fail when shibboleth is active.